src/mm-mapp.js

```diff
@@ -72,22 +72,40 @@ export class MmMapp extends HTMLElement {
   constructor() {
     // Always call super first in constructor
     super();
-    // SUPER IMPORTANT TO SET THIS UP FIRST SO THAT LEAFLET ISN'T WORKING WITH
-    // A HEIGHT=0 BOX BY DEFAULT.
```

Comment:
Worried that this is gone

Reply:
This is the same as https://github.com/Maps4HTML/Web-Map-Custom-Element/pull/108#discussion_r453887570.